geode-core/src/distributedTest/java/org/apache/geode/internal/cache/RegionConcurrentOperationDUnitTest.java

```diff
@@ -131,6 +140,49 @@ public void getOnPreLoadedRegionFromMultipleThreadsReturnSameObject() throws Exc
     assertThat(cacheRule.getCache().getRegion(regionName).size()).isEqualTo(1);
   }
 
+  @Test
+  public void getOnPartitionedRegionFromMultipleThreadsReturnsDifferentPdxInstances()
+      throws Exception {
+    String regionName = getClass().getSimpleName();
+    CacheFactory cacheFactory = new CacheFactory();
+    cacheFactory.setPdxReadSerialized(true);
+    cacheRule.createCache(cacheFactory);
+    InternalCache cache = cacheRule.getCache();
+    cache.setCopyOnRead(true);
+    Region region = cache.createRegionFactory(PARTITION)
```

Comment:
The compiler warning here (and elsewhere in the file) can be avoided by using `Region<Object, Object>`.